### ISSUE TITLE
implicit label

### ISSUE THREAD

**Issue by Aniort** [Tanaguru/webextension#7]
Test A11y-3 implicit labels are false negative, they should be not detected as an error but it's not the case

**Comment by fhalna** [comment]
Fix
